src/bin/cratesfyi.rs

```diff
@@ -446,11 +446,17 @@ impl DatabaseSubcommand {
 
             Self::Delete {
                 command: DeleteSubcommand::Version { name, version },
-            } => db::delete_version(&mut *ctx.conn()?, &*ctx.storage()?, &name, &version)
-                .context("failed to delete the crate")?,
+            } => db::delete_version(
+                &mut *ctx.conn()?,
+                &*ctx.storage()?,
+                &*ctx.config()?,
+                &name,
+                &version,
+            )
+            .context("failed to delete the crate")?,
             Self::Delete {
                 command: DeleteSubcommand::Crate { name },
-            } => db::delete_crate(&mut *ctx.conn()?, &*ctx.storage()?, &name)
+            } => db::delete_crate(&mut *ctx.conn()?, &*ctx.storage()?, &*ctx.config()?, &name)
```

Comment:
maybe just pass in the whole context at this point?

Reply:
good point, changed.